docker-compose.yml

```diff
@@ -0,0 +1,295 @@
+    depends_on:
+      - kafka
+      - postgres
+  channel:
+    build:
+      context: ../Jugalbandi-Manager
+      dockerfile: ./channel/Dockerfile
+    environment:
+      - POSTGRES_DATABASE_NAME=${POSTGRES_DATABASE_NAME}
+      - POSTGRES_DATABASE_USERNAME=${POSTGRES_DATABASE_USERNAME}
+      - POSTGRES_DATABASE_PASSWORD=${POSTGRES_DATABASE_PASSWORD}
+      - POSTGRES_DATABASE_HOST=${POSTGRES_DATABASE_HOST}
+      - POSTGRES_DATABASE_PORT=${POSTGRES_DATABASE_PORT}
+      - KAFKA_BROKER=${KAFKA_BROKER}
+      - KAFKA_USE_SASL=${KAFKA_USE_SASL}
+      - KAFKA_PRODUCER_USERNAME=${KAFKA_PRODUCER_USERNAME}
+      - KAFKA_PRODUCER_PASSWORD=${KAFKA_PRODUCER_PASSWORD}
+      - KAFKA_LANGUAGE_TOPIC=${KAFKA_LANGUAGE_TOPIC}
+      - KAFKA_FLOW_TOPIC=${KAFKA_FLOW_TOPIC}
+      - KAFKA_CHANNEL_TOPIC=${KAFKA_CHANNEL_TOPIC}
+      - KAFKA_CONSUMER_USERNAME=${KAFKA_CONSUMER_USERNAME}
+      - KAFKA_CONSUMER_PASSWORD=${KAFKA_CONSUMER_PASSWORD}
+      - AZURE_STORAGE_ACCOUNT_URL=${AZURE_STORAGE_ACCOUNT_URL}
+      - AZURE_STORAGE_ACCOUNT_KEY=${AZURE_STORAGE_ACCOUNT_KEY}
+      - AZURE_STORAGE_CONTAINER=${AZURE_STORAGE_CONTAINER}
+      - ENCRYPTION_KEY=${ENCRYPTION_KEY}
+      - STORAGE_TYPE=${STORAGE_TYPE}
+      - PUBLIC_URL_PREFIX=${PUBLIC_URL_PREFIX}
+      - WA_API_HOST=${WA_API_HOST}
+    depends_on:
+      - kafka
+      - postgres
+    volumes:
+      - ./media:/mnt/jb_files
+  frontend:
+    build:
+      context: ../Jugalbandi-Manager
+      dockerfile: ./frontend/Dockerfile
+    ports:
+      - "4179:4179"
+    environment:
+      - VITE_SERVER_HOST=${SERVER_HOST}
+    # depends_on:
+    #   - api
+  
+  postgres:
+    image: ankane/pgvector
+    ports:
+      - "5432:5432"
+    environment:
+      #   POSTGRES_USER: postgres
+      POSTGRES_PASSWORD: postgres
+    #   POSTGRES_DB: postgres
+    volumes:
+      - postgres_data:/var/lib/postgresql/data
+      - ./combined_dump.sql:/docker-entrypoint-initdb.d/dump.sql
+  postgres-adminer:
+    image: adminer
+    restart: always
+    ports:
+      - 8080:8080
+  metabase:
+    image: metabase/metabase:latest
+    ports:
+      - "9000:9000"
+  kafka:
+    image: docker.io/bitnami/kafka:3.6
+    ports:
+      - "9092:19092"
+    volumes:
+      - "kafka_data:/bitnami"
+      - ./scripts/create-all-topics.sh:/usr/bin/create-topics.sh
+    command: "sh -c './run.sh & /usr/bin/create-topics.sh && tail -f /dev/null'"
+    environment:
+      - KAFKA_CFG_NODE_ID=0
+      - KAFKA_CFG_PROCESS_ROLES=controller,broker
+      - KAFKA_CFG_CONTROLLER_QUORUM_VOTERS=0@kafka:9093
+      # Listeners
+      - KAFKA_CFG_LISTENERS=PLAINTEXT://:9092,CONTROLLER://:9093,EXTERNAL://:19092
+      - KAFKA_CFG_ADVERTISED_LISTENERS=PLAINTEXT://kafka:9092,EXTERNAL://${JBHOST}:9092
+      - KAFKA_CFG_LISTENER_SECURITY_PROTOCOL_MAP=CONTROLLER:PLAINTEXT,PLAINTEXT:PLAINTEXT,EXTERNAL:PLAINTEXT
+      - KAFKA_CFG_CONTROLLER_LISTENER_NAMES=CONTROLLER
+      - KAFKA_CFG_INTER_BROKER_LISTENER_NAME=PLAINTEXT
+  # PwR Studio
+  server:
```

Comment:
Arrange these correctly:

JB Manager service
Studio service
db
kafka

(others are not required)